application/modules/reports/models/Mdl_reports.php

```diff
@@ -188,6 +188,25 @@ public function invoice_aging()
         return $this->db->get('ip_clients')->result();
     }
 
+    public function invoices_per_client($from_date = null, $to_date = null)
+    {
+        $from_date = date_to_mysql($from_date);
+        $to_date = date_to_mysql($to_date);
+
+        $this->db->select('*');
```

Comment:
Wouldn't that become a heavy query when I have 1000 clients and 10 invoices per client?
How many fields of rhe client are we using?
It's using the format_client function, so maybe `client_name`, their address, etc